Minio/V4Authenticator.cs

```diff
@@ -403,10 +403,10 @@ private static string GetCanonicalHost(Uri url)
             switch (contentKind)
             {
                 case "application/x-www-form-urlencoded":
-                {
-                    isFormData = true;
-                    cntntByteData = requestBuilder.Request.Content.ReadAsByteArrayAsync().Result;
-                }
+                    {
+                        isFormData = true;
+                        cntntByteData = requestBuilder.Request.Content.ReadAsByteArrayAsync().Result;
+                    }
```

Comment:
This code formatting is wrong - the previous alignment is correct, please run 

```
dotnet regitlint -f commits -a HEAD
```

locally.

Reply:
Done